### PR TITLE
Add Twitter feed

### PR DESCRIPTION
**Pull request by ajantsch**
Closes https://github.com/gesund-zusammen/gesund-zusammen.github.io/issues/48

Screenshot 2020-04-07 13 36 59

### CODE REVIEW

**Comment by manzke** [comment]
this is how it looks like for me
Screenshot 2020-04-07 at 13 44 39

File: src/components/layouts/Shared/Footer.tsx [hunk 1]
```diff
@@ -92,26 +71,17 @@ class Footer extends React.PureComponent<WithTranslation, {}> {
 const FooterWrapper: AnyStyledComponent = styled(Box)`
   font-family: "Open Sans", "Helvetica", "Arial", sans-serif;
   color: #ffffff;
-  padding: 2rem 0 1rem 0;
-  background-color: #ccedf7;
+  padding: 2rem 0 0 0;
+  background-color: #003269;
```
Comment (manzke) [inline src/components/layouts/Shared/Footer.tsx:75]: please revert color change

File: src/components/layouts/Shared/Social.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                </Typography>
+                <Typography variant="body1" style={{ color: "#ffffff" }}>
+                  {this.props.t("footer.claim")}
+                </Typography>
+              </SocialContent>
+              <FooterShareBox />
+            </Grid>
+            <Grid item xs={12} sm={6}>
+              <TwitterTimelineEmbedWrapper>
+                <Typography variant="h2" style={{ color: "#ffffff" }}>
```
Comment (manzke) [inline src/components/layouts/Shared/Social.tsx:37]: do we need the title if the embedded one spreads twitter everywhere as well?

**Comment by ajantsch** [comment]
Screenshot 2020-04-07 14 00 38

**Pull request merged**